systems/WikibaseClient/diagrams/05-entitychangenotifications-usage.drawio.svg

```diff
@@ -0,0 +1,307 @@
+                                Persistence
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="455" y="35" fill="#000000" font-family="Helvetica" font-size="15px" text-anchor="middle">
+                    Persistence
+                </text>
+            </switch>
+        </g>
+        <rect x="260" y="0" width="110" height="60" fill="#e1d5e7" stroke="#9673a6" pointer-events="all"/>
+        <g transform="translate(-0.5 -0.5)">
+            <switch>
+                <foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 108px; height: 1px; padding-top: 30px; margin-left: 261px;">
+                        <div style="box-sizing: border-box; font-size: 0; text-align: center; ">
+                            <div style="display: inline-block; font-size: 15px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">
+                                Acumulation
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="315" y="35" fill="#000000" font-family="Helvetica" font-size="15px" text-anchor="middle">
+                    Acumulation
+                </text>
+            </switch>
+        </g>
+        <rect x="535" y="0" width="110" height="60" fill="#e1d5e7" stroke="#9673a6" pointer-events="all"/>
+        <g transform="translate(-0.5 -0.5)">
+            <switch>
+                <foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 108px; height: 1px; padding-top: 30px; margin-left: 536px;">
+                        <div style="box-sizing: border-box; font-size: 0; text-align: center; ">
+                            <div style="display: inline-block; font-size: 15px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">
+                                Huh?
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="590" y="35" fill="#000000" font-family="Helvetica" font-size="15px" text-anchor="middle">
+                    Huh?
```

Comment:
Looks a bit funny to keep it here, but combined with the warning it's probably ok as an indicator of something that needs to be cleaned up :D